pages/projects.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Projects</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700&family=Montserrat:ital,wght@0,400;1,700&display=swap" rel="stylesheet">
+  <link href="//db.onlinewebfonts.com/c/0fddb08144cea533dcd3d39388b7ffbe?family=GrandSlang+Roman" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+
+  <header>
+    <nav class="block">
+        <ul class="home-nav">
+            <li><a href="home.html">Home</a></li>
+            <li><a href="about.html">About</a></li>    
+            <li><a href="projects.html">Projects</a></li>    
+            <li><a href="contact.html">Contact</a></li>   
+        </ul>
+    </nav>
+  </header>
+
+  <section id='title'>
+    <h1 >
+      Projects
+    </h1>
+  </section>
+
+  <div class="gallery-container">
+    <figure class=" gallery-item">
+      <a href="https://github.com/Anagabsoares/retro-video-store">
+        <img class="image-projects" src="../images/image_1.png" alt="Image 1">
```

Comment:
"Image 1" doesn't tell me what the image is a picture _of_:

```suggestion
        <img class="image-projects" src="../images/image_1.png" alt="A screenshot of the retro video store github repository.">
```